lib/index.js

```diff
@@ -65,7 +65,7 @@ function VM (opts = {}) {
     this.stateManager = new StateManager({ trie, common: this._common })
   }
 
-  this.blockchain = opts.blockchain || fakeBlockchain
+  this.blockchain = opts.blockchain || new Blockchain({ common: this._common })
```

Comment:
Just for clarification: since the `db` parameter is not provided, `Blockchain` will create an in-memory leveldb instance to store blocks and their metadata.